cli-hangman.js

```diff
@@ -0,0 +1,96 @@
+var inquirerUserPrompt = require("inquirer");
+var Word = require("./word.js");
+
+var tries = 0;
+var lettersArray = [];
+var currentString = '';
+var currentWord;
+var lettersGuessed = [];
```

Comment:
I would avoid global variables wherever possible now that we are starting to modularize our code more. Instead try this.
```javascript
var state = {
  tries = 0;
  lettersArray = [];
  currentString = '';
  currentWord;
  lettersGuessed = [];
}